src/scss/abstracts/_mixins.scss

```diff
@@ -1 +1,11 @@
 // Mixins.
+@mixin focus-style($c) {
+	a:focus,
+	a:hover,
+	#defaultContainer header .site-nav-wrapper .site-nav nav a:focus,
+	#defaultContainer header .site-nav-wrapper .site-nav nav a:hover,
+	#defaultContainer .content-wrapper .tags-info .tags a:focus,
+	#defaultContainer .content-wrapper .tags-info .tags a:hover, {
+		border: rem(2) solid $c;
+	}
```

Comment:
@greatislander I came up with this solution to get by the linting error that was appearing during check-in. Let me know what you think

Reply:
Awesome, that was what I was going to suggest!